tasks/main.yml

```diff
@@ -1,2 +1,58 @@
----
-# tasks file for ansible-newrelic-inode-plugin
+# ansible-playbook inode_monitor.yml -i vendor/ec2.py -vvvv --ask-vault-pass
+- name: Add inode monitoring to servers
+  hosts: "{{ target_hosts | default('tag_class_app:&tag_app_environment_staging:&tag_app_name_insurance')}}"
+  become: true
+  user: ubuntu
+  vars_files:
+    - vault.yml
+    - vars.yml
+  tasks:
+   - name: Install inode monitor on server
+     git:
+       repo: https://github.com/Foild/newrelic-inode-plugin.git
+       dest: /usr/local/bin/newrelic-inode-plugin
```

Comment:
this `/usr/local/bin/newrelic-inode-plugin` is getting use on different places can be a role deafult?